spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/ConfigCommands.java

```diff
@@ -256,12 +283,11 @@ public String info() {
 		return sb.toString();
 	}
 
-
 	@Override
 	public void onApplicationEvent(ApplicationReadyEvent event) {
 		//Only invoke if the shell is executing in the same application context as the data flow server.
 		if (!initialized) {
-			target(this.serverUri, this.userName, this.password);
+			target(this.serverUri, this.userName, this.password, isSkipSslValidation(this.environment));
```

Comment:
I meant the same pattern as what was there before for uri, username and password (a member field)